astro/cli/astro/cmd/display.go

```diff
@@ -32,7 +32,7 @@ import (
 
 // printExecStatus takes channels for status updates and exec results
```

Comment:
warning: cyclomatic complexity 16 of function (*AstroCLI).printExecStatus() is high (> 15) (gocyclo)